docs/docsite/rst/reference_appendices/special_variables.rst

```diff
@@ -28,6 +28,18 @@ ansible_limit
 ansible_loop
     A dictionary/map containing extended loop information when enabled via ``loop_control.extended``
 
+ansible_parent_role_name
+    When the current role is being executed by means of an include_role action, this variable contains the role name that included this role. Otherwise, this variable is not defined.
+
+ansible_parent_role_names
+    Similar to ``ansible_parent_role_name``, only contains a list of all parent roles, with the most recent role (i.e. the role that included this role) being the first item in the list.
```

Comment:
I'm a bit confused by this. When a role has multiple parent roles, is the relationship a tree (Role A includes Role B, Role B includes Role C . . . Role C's parents are B first and then A)? Or will this show if one role is (mistakenly?) included more than once by different roles (Role A includes Role M, Role G also includes Role M . . . Role M's parents are G and A in order of the playbook)? This is more for my information than a comment on the PR or the docs.

Reply:
As mentioned on IRC; I expanded these sections a bit to explain the order of the ``ansible_parent_role_names`` and ``ansible_parent_role_paths`` variables. Since there is no area in the documentation about how to deal with _being included_ by other roles, there isn't really a spot for this PR to attach itself to. It would likely make more sense that a separate chapter be made specifically about creating reusable roles and how to deal with being included by other roles.